zapisy/apps/theses/forms.py

```diff
@@ -47,7 +47,9 @@ class Meta:
         label="Przypisani studenci",
         widget=forms.SelectMultiple(attrs={'size': '10'}))
     status = forms.ChoiceField(choices=ThesisStatus.choices, label="Status")
-    reserved_until = forms.DateField(widget=forms.TextInput(attrs={'type': 'date'}),
+    reserved_until = forms.DateField(help_text="Jeżeli przypiszesz do pracy studentów,\
+                                     uzupełnij również datę rezerwacji.",
```

Comment:
W Pythonie jest taki fajny trick na łączenie stringów.
```suggestion
    reserved_until = forms.DateField(help_text="Jeżeli przypiszesz do pracy studentów, "
                                     "uzupełnij również datę rezerwacji.",
```